app/templates/components/pc-nav.hbs

```diff
@@ -100,6 +100,17 @@
           {{t "djinquiry.title"}}
         </LinkTo>
       </li>
+      <li>
```

Comment:
I'm not sure about the placement of the sign up being under "radio". I was thinking it would have been aligned next to the "login" button on the Chat page.

Reply:
Yeah I'm not sure how I feel about it either. I was thinking maybe there are too many buttons down there already....please someone who is a designer god help us :rofl: